### PR TITLE
Update update-phone-number-and-email-address.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to reflect the new O365 setting to update alternate email and phone number for admins.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2374

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/manage/update-phone-number-and-email-address.md [hunk 1]
```diff
@@ -37,13 +37,13 @@ If you're looking for how to change your company's profile information, such as
...
-Use the **Personal info** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
+Use the **Settings and Privacy** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
   
 1. Browse to the Microsoft 365 admin center.
 
-2. In the header, select your profile icon \> **My account** \> **Personal info**.
+2. In the header, select your profile icon \> **My account** \> **Settings and Prvicay**.
 
-3. In the **Contact details** section, select Edit. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
+3. In the **Contact Preferences** section, select Manage and expand Contact preferences. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/update-phone-number-and-email-address.md]:
```suggestion
3. In the **Contact Preferences** section, select **Manage and expand Contact preferences**. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
```

File: microsoft-365/admin/manage/update-phone-number-and-email-address.md [hunk 2]
```diff
@@ -37,13 +37,13 @@ If you're looking for how to change your company's profile information, such as
   
 ## To update your name, phone number, and email address
 
-Use the **Personal info** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
+Use the **Settings and Privacy** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
   
 1. Browse to the Microsoft 365 admin center.
 
-2. In the header, select your profile icon \> **My account** \> **Personal info**.
+2. In the header, select your profile icon \> **My account** \> **Settings and Prvicay**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/update-phone-number-and-email-address.md]:
```suggestion
2. In the header, select your profile icon \> **My account** \> **Settings and Privacy**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuA Please merge this PR. Thanks.

**Pull request merged**